posts/2019-01-30-cats-ecosystem-community-survey-results.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Cats Ecosystem Community Survey 2018 Results
+
+meta:
+  nav: blog
+  author: kailuowang
+  pygments: true
+
+---
+
+Overall we received 588 responses over the course of 30 days. These feedbacks are essential for us to make informed decisions on our 2019 plan. Thank you, everyone, who participated. 
+
+As promised, here are the results, as well as some quick read from us. 
+
+<br /><br /><br />
+### Q: How long have you been using the Cats ecosystem (including Cats and Cats ecosystem libraries)?
+#### Results:
+![1](/img/media/2018-survey/5WuSHVP.png)
+
+#### Our read:
+46% of the respondents are relatively new (less than 1 year) users. Welcome!
+
+<br /><br /><br />
+### Q: Do you feel welcome in the Cats ecosystem community?
+
+#### Results:
+![2](/img/media/2018-survey/JjK3muU.png)
+
+#### Our read:
+83% of users gave the 4+ ratings on the welcomeness of the community. This is a promising sign. We should aim for having more 5 ratings (currently at 42.5%) and less 3- ones. 
+
+<br /><br /><br />
+### Q: How can we be more welcoming?
+
+#### Responses summary: 
+We received 92 responses on this free form question. The vast majority, 90+%, of them suggested that more documentation, tutorial, and real-world examples will help them feel more welcome. Some of them suggested that the learning curve could be less intimidating to them if there are more introductory resources. 
+
+#### Our read: 
+
+The learning curve for Cats ecosystem libraries could be steep for people new to pure functional programming. This is by far the most impactful area for us to work on to be more inclusive. 
+
+<br /><br /><br />
+### Q: In what types of projects do you primarily use the Cats ecosystem?
+
+#### Results:
+![3](/img/media/2018-survey/uqUxPWf.png)
+
+#### Our read:
+83% of respondents are using Cats ecosystem libraries in production applications. We are honored to have that trust. In the meantime, it's a great responsibility for us to maintain stability and robustness. 
+
+<br /><br /><br />
+### Q: In which application domain do you primarily use the Cats ecosystem?
+
+#### Results:
+![4](/img/media/2018-survey/cOhZ8W3.png)
+
+The free form `other` responses are omitted for the sake of conciseness. 
+
+
+<br /><br /><br />
+### Q: If you use Cats directly, how is your overall experience using Cats?
+
+#### Results:
+![5](/img/media/2018-survey/q2WjZ1d.png)
+
+
+### Q: What would significantly improve your experience using Cats?
+#### Results:
+![5](/img/media/2018-survey/3ne70PU.png)
+
+The free form `other` responses can be summarized as 
+1. better IDE support 10 (1.8%)
+2. better imports 7 (1.3%)
+3. better integration with other libs 3 (0.6%)
+
+
+#### Our read:
+Overall the experience is mostly positive for our users. 73% of them believe there is space for improvements - they gave a rating less than 5. Again, more documentation and training will help most users. There are also 25% of users are looking for more features. 
```

Comment:
`There are also 25% of users are looking` is awkward, how about `25% of users are looking`